packages/opencensus-web-instrumentation-zone/src/interaction-tracker.ts

```diff
@@ -26,42 +28,95 @@ export type AsyncTask = Task & {
   data: AsyncTaskData;
   eventName: string;
   target: HTMLElement;
+  // Allows access to the private `_zone` property of a Zone.js Task.
+  _zone: Zone;
 };
 
 export class InteractionTracker {
+  // Store the Zones indexed by unique ID in order to check current zones.
+  // private readonly tracingZones: { [index: string]: Zone } = {};
```

Comment:
Could you remove this and the other commented out code below if it's no longer needed?